Makefile

```diff
@@ -10,7 +10,7 @@ build: $(BUILDDIR)/
 build: BUILD_ARGS=-o=$(BUILDDIR)
 
 $(BUILD_TARGETS):
-	go $@ -mod=readonly $(BUILD_FLAGS) $(BUILD_ARGS) ./...
+	go $@ $(BUILD_FLAGS) $(BUILD_ARGS) ./...
```

Comment:
Isn't it a good idea to keep `-mod=readonly` to avoid potential mod updates after building?

Reply:
Sure. if you recommend that. I've removed because I haven't seen it much used it.

Reply:
I believe that without that flag you could potentially update go mod information during build which would lead to file changes in the local repo. Leaving the flag would avoid that making mod updates something that would be done explicitly when required.

Reply:
I've added it back